cg/clients/freshdesk/models.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class Ticket(BaseModel):
+    """Freshdesk ticket."""
```

Comment:
Not sure what the custom is here. Should one have separate models for request and response? Or just have a big one with everything optional so which can be used both as input and output?

The reason for unifying them was because the overlap was so large

Reply:
What does the response add? Timestamps for creation? And are they left blank if something goes wrong? You could consider having a parent class with two child classes.

Reply:
I would make a model with the required fields for a CreateTicket model that we actually will use. We can then still adjust if we need to add something later on. 

I think having all these fields could potentially be confusing.

Reply:
So for the creation only about 5 fields are used, but the objects that are returned contain a lot. But importantly an id and timestamps.

I agree that most fields are unnecessary. Would you both prefer two classes TicketCreate and TicketResponse which only contain the fields we intend to use, and add more fields if we decide to use them?

Reply:
I agree, sort of similar to the request/response from DDN.

Reply:
Addressed in [3dca545](https://github.com/Clinical-Genomics/cg/pull/3317/commits/3dca545ffb56567f9685a269e81d154581de9f88)